src/inflector.erl

```diff
@@ -178,14 +178,14 @@ singulars() ->
       {"s$",                     ""      } ] ++ reversed_irregulars().
 
 irregulars() ->
-    [{"move",   "moves"   },
-     {"sex",    "sexes"   },
-     {"child",  "children"},
-     {"man",    "men"     },
-     {"person", "people"  }].
+    [{"move$",   "moves"   },
+     {"sex$",    "sexes"   },
+     {"child$",  "children"},
+     {"man$",    "men"     },
+     {"person$", "people"  }].
 
 reversed_irregulars() ->
-    F = fun ({A, B}) -> {B, A} end,
+    F = fun ({A, B}) -> {B ++ "$", re:replace(A, "\\$", "", [{return, list}])} end,
```

Comment:
Line 188 is too long:     F = fun ({A, B}) -> {B ++ "$", re:replace(A, "\\$", "", [{return, list}])} end,.